homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -76,8 +77,15 @@ def icon(self):
         """Icon to use in the frontend, if any."""
         return ICON
 
+    @property
+    def device_state_attributes(self):
+        """Return device specific state attributes."""
+        if self._type == "production":
+            return self._attributes
```

Comment:
Add an explicit return None below this.

Reply:
I ended up moving the conditional to the update method instead is that OK? Also, any input on the questions above?